### PR TITLE
Build docker component

### PR DESCRIPTION
**Pull request by rafaelhernandez**
@wyattfry could you please review this PR? Thanks!

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+version: "2"
```
Comment (wyattfry) [inline docker-compose.yml]: consider using version 3, removing `links` replace with `depends_on`

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    build: .
+    ports:
+      - "3002:3002"
+    links:
+      - mongo
+  mongo:
+    container_name: mongo
+    image: mongo:3.4.13
+    ports:
+      - "27027:27017"
```
Comment (wyattfry) [inline docker-compose.yml:18]: must you expose the db to the host system?

**Review by wyattfry — COMMENTED** [review]
a few non-essential comments

File: docker-compose.yml (hunk 2 above)
Comment (rafaelhernandez) [reply docker-compose.yml:18]: It is useful for testing purposes. Allows for accessing the database using mongo client.

**Pull request merged**